pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/WorkerConfig.java

```diff
@@ -123,6 +122,8 @@
         private String javaInstanceJarLocation;
         private String pythonInstanceLocation;
         private String logDirectory;
+        // the directory for dropping extra function dependencies
+        private String extraFunctionDependenciesLocation;
```

Comment:
Should we name 'extraFunctionDependenciesLocation' and 'extraFunctionDependenciesDir' the same?

Reply:
I make them different, because the behavior in process mode and k8s is different. Because k8s has a pulsarRootDir, but process mode doesn’t. If we use the same name, it will become confused.

However I am open to any change, if you think we should use same variable.